plutus-contract/test/Spec/golden/traceOutput - pubKeyTransactions.txt

```diff
@@ -1,65 +1,65 @@
 Slot 00000: TxnValidate 98d5fbcefe21113b3f0390c1441e075b8a870cc5a8fa2a56dcde1d8247e41715
 Slot 00000: SlotAdd Slot 1
-Slot 00001: W1bc5f27: InsertionSuccess: New tip is Tip(slot= Slot 1, blockId= BlockId(-3933910562423127441), blockNo= 0). UTxO state was added to the end.
```

Comment:
@kk-hainq - do you know why these are changing?

Reply:
A newer `hashable` version changed the default `Hashable` instance of `ByteString`:
https://github.com/haskell-unordered-containers/hashable/blob/747ff1b58b721fbf355ea1b1a9b66f531bf966e9/CHANGES.md?plain=1#L31-L34

Which affects `PlutusTx.Builtins.Internal.BuiltinByteString`, then `Plutus.V1.Ledger.TxId.TxId`, then the `blockId` in `Plutus.Trace.Emulator.System.appendNewTipBlock`:
https://github.com/input-output-hk/plutus-apps/blob/8574e9678d7288d3cc3bdafdfa425f5e096b77d4/plutus-contract/src/Plutus/Trace/Emulator/System.hs#L146-L150

Which eventually arrives at this log:
https://github.com/input-output-hk/plutus-apps/blob/8574e9678d7288d3cc3bdafdfa425f5e096b77d4/plutus-chain-index-core/src/Plutus/ChainIndex/Emulator/Handlers.hs#L189

Either way, good catch! The numeric, negative even block ids indeed look weird. I've replaced the ad-hoc hash function with the more appropriate `blockId` function in the latest commit: https://github.com/input-output-hk/plutus-apps/pull/269/commits/76358f5c55ea5b7afa31108f318fa32a5bbd4228.